### PR TITLE
Add new OCP 4.12 version to the DAGs

### PR DESCRIPTION
**Pull request by krishvoor**
Signed-off-by: Krishna Harsha Voora <krvoora@redhat.com>

### CODE REVIEW

File: dags/openshift_nightlies/config/install/hypershift/sdn.json [hunk 1]
```diff
@@ -7,12 +7,12 @@
     "managed_channel_group": "nightly",
     "managed_ocp_version": "latest",    
     "openshift_worker_count": 9,
-    "openshift_network_type": "OpenShiftSDN",
```
Comment (mukrishn) [inline dags/openshift_nightlies/config/install/hypershift/sdn.json:10]: this install config is meant for sdn type only, so instead we remove this dag.

File: dags/openshift_nightlies/manifest.yaml [hunk 2]
```diff
@@ -126,7 +126,7 @@ platforms:
 
 # Do not program concurrent builds of ROSA/ROGCP/ARO
   rosa:
-    versions: ["4.10", "4.11"]
```
Comment (mukrishn) [inline dags/openshift_nightlies/manifest.yaml:129]: Also remove `4.10`

Comment (krishvoor) [reply dags/openshift_nightlies/manifest.yaml:129]: Done

File: dags/openshift_nightlies/config/install/hypershift/sdn.json (hunk 1 above)
Comment (krishvoor) [reply dags/openshift_nightlies/config/install/hypershift/sdn.json:10]: Agreed, removed

File: dags/openshift_nightlies/manifest.yaml [hunk 3]
```diff
@@ -1,8 +1,4 @@
 versions:
-- version: "4.10"
```
Comment (mukrishn) [inline dags/openshift_nightlies/manifest.yaml:2]: this should not be removed, instead previous to be `4.11`, stable to `4.12` and next to `4.13` but Managed Service cluster has nothing to do with these `versions` so I'd leave this to self-managed team handle this rollout.

File: dags/openshift_nightlies/manifest.yaml [hunk 4]
```diff
@@ -180,7 +176,7 @@ platforms:
           benchmarks: upgrade.json
 
   rogcp:
-    versions: ["4.10", "4.11"]
+    versions: ["4.10", "4.11", "4.12"]
```
Comment (mukrishn) [inline dags/openshift_nightlies/manifest.yaml:179]: missed this one

File: dags/openshift_nightlies/manifest.yaml (hunk 3 above)
Comment (mukrishn) [reply dags/openshift_nightlies/manifest.yaml]: @krishvoor lets keep this and remove 4.10 from others too

**Review by mukrishn — APPROVED** [review]

**Pull request merged**